ocs_ci/deployment/deployment.py

```diff
@@ -752,6 +753,28 @@ def setup_local_storage():
         timeout=600
     ), "Local storage operator did not reach running phase"
 
+    # Add RDM disk for vSphere platform
+    platform = config.ENV_DATA.get('platform').lower()
+    lso_type = config.DEPLOYMENT.get('type')
+    if platform == constants.VSPHERE_PLATFORM:
+        # Types of LSO Deployment
+        # Importing here to avoid circular dependancy
+        from ocs_ci.deployment.vmware import VSPHEREBASE
```

Comment:
what about turn this function into method in Deployment base class? Than you will not have to import here from VSPHEREBASE and instantiate VSPHEREBASE.  As we will alway have the proper deployment instantiated based on platform, here you can just continue with condition below and just call self.add_rdm_disks(). WDYT?

Reply:
Ack. I will take it up in next PR